src/world/Territory/Land.cpp

```diff
@@ -242,18 +242,16 @@ Sapphire::Land::InvMaxItemsPair Sapphire::Land::getInventoryItemMax() const
 {
   switch( m_size )
   {
+    default:
```

Comment:
return already breaks out of the case implicitly, and thus `break` is unnecessary
same for all `break`s below